astropy/units/quantity_helper/function_helpers.py

```diff
@@ -1057,6 +1064,8 @@ def eig(a, *args, **kwargs):
     return (a.value,)+args, kwargs, (a.unit, dimensionless_unscaled), None
 
 
+# ===================================================================
```

Comment:
Fine to add but maybe more like `# ===== np.lib.recfunctions ====`?

Reply:
Hmm, marked as resolved but not actually done!?

Reply:
oops. Forgot to push.

Reply:
Pushed.